src/Resources/SubFormRelationManager.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Filament\Resources;
+
+/**
+ * This is a stub class for HasOne/BelongsTo relationships to add forms on a resource page.
+ * It will work largely like the RelationManager, just without the table method.
+ */
+abstract class SubFormRelationManager extends RelationManager
```

Comment:
The benefit of keeping it a novel abstract class is that it prevents a need to add more conditional checks to the `canCrate`, `canAttach` and `canDetach` methods. It also prevents the need for an always empty table method that's required due to extending the `RelationManager`.

Both of these are a benefit as this use case will never need Attach/Detach - nor do we want to invoke any thing related to the default `RelationManager` create process. We just want the EditRecord option essentially so we can embed it into the page.